tests/server/3-document.spec.js

```diff
@@ -100,7 +100,13 @@ describe('Document', () => {
           .set('authorization', adminToken)
           .end((err, res) => {
               res.status.should.equal(200);
-              res.body[1].should.have.value('role', 'regular');
+              for(let i=0; i<res.body.length; i++) {
```

Comment:
### Nitpick

There should be spacing between `operands` and `operators` e.g `i = 0 not i=0`